ci-operator/config/openshift/run-once-duration-override-operator/openshift-run-once-duration-override-operator-main.yaml

```diff
@@ -0,0 +1,55 @@
+base_images:
+  base:
+    name: "4.13"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.19-openshift-4.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.13
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.rhel7
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.13:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
+  to: run-once-duration-override-operator
+promotion:
+  name: "4.13"
+  namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.13"
+  namespace: ocp
+tests:
+- as: verify
+  commands: make verify-gofmt
+  container:
+    from: src
+- as: unit
+  commands: make test-unit
```

Comment:
Should these commands be defined in run-once-duration-operator makefile?

Reply:
They are. Defined through the build-machinery-go.
```sh
$ make help
$ make help
The following make targets are available:
all
build
clean
clean-binaries
clean-controller-gen
clean-yaml-patch
clean-yq
ensure-controller-gen
ensure-imagebuilder
ensure-yaml-patch
ensure-yq
generate
generate-clients
help
image-runoncedurationoverride-operator
images
Makefile
manifests
regen-crd
test
test-e2e
test-unit
update
update-codegen
update-codegen-crds
update-deps-overrides
update-generated
update-gofmt
verify
verify-codegen
verify-codegen-crds
verify-deps
verify-generated
verify-gofmt
verify-golang-versions
verify-golint
verify-govet
```

Reply:
Learned something new, thanks :)